packages/shared/Messages.ts

```diff
@@ -2,3 +2,4 @@ export const MSG_PLAYGROUNDS_STATE_UPDATE = 'playgrounds-state-update';
 export const MSG_NEW_PLAYGROUND_QUERIES_STATE = 'new-playground-queries-state';
 export const MSG_NEW_CATALOG_STATE = 'new-catalog-state';
 export const MSG_NEW_RECIPE_STATE = 'new-recipe-state';
+export const MSG_NEW_LOCAL_MODELS_STATE = 'new-local-models-state';
```

Comment:
why using `new`, it is more likely to be an `updated` since the watcher will detect, deletion, update and new

Reply:
New for New state. Every time the state changes, all the state is sent, and the UI rebuilds itself from scratch with this new state. But the updates from the previous state is not taken into consideration. Hence the `New`.